client/packages/invoices/src/utils.test.ts

```diff
@@ -0,0 +1,35 @@
+import { getNextItemId } from './utils';
```

Comment:
Maybe give this file a more specific name ("invoiceUtils.test.ts"?) so we can target just this test? I ran `yarn test utils` and it ran 5 different tests with "utils" in the name/path.

Reply:
Can't we just use the jest runner extension in VSCode if you only want to run these tests?

Reply:
Oh, I don't think I have that one. I normally use partial filename matching when I run specific tests.

Reply:
Ah its good! Similar to rust analyzer but only for the running/debugging tests part. If you search for this extension: firsttris.vscode-jest-runner